modules/shortcodes/slideshow.php

```diff
@@ -286,14 +306,16 @@ function enqueue_scripts() {
 			 *
 			 * @module shortcodes
 			 *
-			 * @since 2.1.0
+			 * @since  2.1.0
 			 *
 			 * @param array $args
 			 * - string - spinner - URL of the spinner image.
 			 */
-			apply_filters( 'jetpack_js_slideshow_settings', array(
-				'spinner' => plugins_url( '/img/slideshow-loader.gif', __FILE__ ),
-			) )
+			apply_filters(
+				'jetpack_js_slideshow_settings', array(
```

Comment:
will having `apply_filters` on multiple lines break the upcoming developer site @dereksmart?